src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentFactory.java

```diff
@@ -34,7 +34,7 @@
 import hudson.model.TaskListener;
 import hudson.remoting.VirtualChannel;
 import jenkins.security.MasterToSlaveCallable;
-import org.apache.tomcat.jni.Library;
+import io.netty.internal.tcnative.Library;
```

Comment:
I don't  know if it works. As I commented before I feel I need to test it but am unsure of how to test the native code path.

It took quite a lot of investigation to figure out where stuff had moved since 2008, and I might have missed a jump somewhere along the way.